ir/src/main/scala/org/scalajs/core/ir/Trees.scala

```diff
@@ -869,6 +869,37 @@ object Trees {
      */
     final case class Global(path: List[String]) extends JSNativeLoadSpec
 
+    /** Load from a module import.
+     *
+     *  The `module` is the ES module identifier. The `path` is a series of
+     *  nested property names starting from the module object.
+     *
+     *  The path can be empty, in which case the specification denotes the
+     *  module itself (whose exports are available as fields).
+     *
+     *  Any element in the path is a property selection from there. A module
+     *  import info with one path element is importing that particular value
+     *  from the module.
+     *
+     *  Examples:
+     *  {{{
+     *  // import { Bar as x } from 'foo'
+     *  Import("foo", List("Bar"))
+     *
+     *  // import { Bar as y } from 'foo'
+     *  // y.Baz
+     *  Import("foo", List("Bar", "Baz"))
+     *
+     *  // import * as x from 'foo' (whole module import)
+     *  Import("foo", Nil)
+     *
+     *  // import x from 'foo' (default import)
+     *  Import("foo", List("default"))
```

Comment:
Yes, we'd need to do this. Otherwise we won't have the duality as in the article:

```javascript
// ES6 equivalent of `var colors = require("colors/safe");`
import colors from "colors/safe";
```

What I do not like about this, is that it relies on magic names. There will be **no** way to fully import a CommonJS module that has a member named `default`. So a later addition to the library might even silently break existing code...

The alternative is of course, that in Scala.js we do make a distinction between the default import and importing `default`. But that means more special casing and a terrible semantics based on legacy, once CommonJS modules are not around anymore.

OTOH it might be a good idea to have `@JSImport` without a second argument be the default import. I guess this mostly depends how modules end up exporting their stuff.
To be safe, we maybe should not allow a single argument for now, so we can define it's semantics later?